fsh/ig-data/input/pages/3_howtoJoin.md

```diff
@@ -39,6 +39,15 @@ To fill the Store of the Bridgehead you need access to clinical data from your *
 The Store of the Bridgehead stores data as **HL7 FHIR® Bundles**. To import data into the Store, they must be harmonised and converted. Each site must build an ETL-process (extract, transform, load) to harmonise the structure of the data and translate them to fit predefined FHIR® profiles.
 You can find our FHIR® profile for the basic data set [here](https://simplifier.net/bbmri.de) and for the oncology data set [here](https://simplifier.net/oncology).
 
+6. Definition of "Sample": 
+A sample, as undestood by the Sample Locator, should satisfy the following conditions:
+
+* Samples for which all parameters are identical are considered to be a single sample and will counted only once - or- may only be loaded once into the store. & 
+* Samples that differ in at least one parameter (central or not) are considered to be different samples and are both counted - or uploaded to the store.
+
+7. Directory collections: 
+If you are participating in the BBMRI Directory, you should create or assign a single collection specifically for use with the Locator. This is because, right now, the Locator can only handle a single collection.  If you only have a single collection, or if there is only one collection that you wish to make available to the Locator, then you should use that collection.  However, if you have samples across multiple collections, then you will need to create a new super-collection to be used with the Locator. Your existing collections will then become sub-collections of the new Locatro collection.
```

Comment:
I think, we don't have hierarchies of collections right now. So I would not talk about super- and/or sub-collections. Also a sample can be only assigned to one collection. 

For the EXLIQUID project, we like to assign certain samples to an EXLIQUID collection. That would go against the fact, that we would like to have all samples in only one collection. We have to discuss that further.

Reply:
Ok, after an internal discussion, that special collection for the negotiation doesn't need to be used for the samples. So instead of saying that the Locator can store and assign only only one collection to the samples, it would be better to say, that it only presents the sample counts only as a summed total and that one needs to mark one collection as the collection the presented counts should be communicated with the negotiator. 

On the other hand, the fact that the locator doesn't groups the counts by collection, is only a German thing. Wouldn't we need the individual counts for the European locator anyway?

Reply:
Re. your second point, yes, the end goal is to allow multiple collections. This text is a stopgap, that will serve unti that goal is achieved. Then we will revise the text appropriately.